modules/core/04-channel/keeper/handshake.go

```diff
@@ -339,7 +339,8 @@ func (k Keeper) WriteOpenAckChannel(
 }
 
 // ChanOpenConfirm is called by the counterparty module to close their end of the
-//  channel, since the other end has been closed.
+//
```

Comment:
Could we please remove all the white spaces that have been added? I think gofumpt formatted it like this because there was a single empty space at the beginning of the line...